FableAnniversary.asl

```diff
@@ -43,9 +44,22 @@ startup {
 }
 
 isLoading {
-	return current.isLoading > 0;
+	if(current.isLoading > 0){
+		return current.isLoading > 0;
+	}
+
+	if(current.isLoadingSave > 0){
+		return current.isLoadingSave > 0;
+	}
+
+	if(current.isLoading == 0 && current.isLoadingSave == 0){
+		return current.isLoading > 0;
+		return current.isLoadingSave > 0;
+	}
 }
 
 split {
-	return old.questsCompleted < current.questsCompleted && settings["split"+current.questsCompleted];
-}
\ No newline at end of file
+	if(current.isLoadingSave == 0){

```

Comment:
Seems redundant/breaking - are there any scenarios in which your quest completion count increases, you are midload, and you *don't* want to split?

Reply:
I spoke with Litigacy elsewhere and he showed me a new scenario in the run where this condition becomes necessary.